setup/installer/installer_runner.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Set up the gcloud environment and Forseti prerequisites.
+""" GCP Installer runner.
```

Comment:
The name `installer_runner.py` might be better as just `installer.py`.

Reply:
That creates a conflict between `setup/installer.py `and `setup/installer/installer_runner.py`